README.md

```diff
@@ -372,6 +372,8 @@ I created [a page in the WIKI](https://github.com/raomin/ESPAltherma/wiki/Inform
 
 No, ESPAltherma supports only Altherma protocol. Other (AC only) units also have a serial port but using other protocols that would require extra reverse engineering to be implemented.
 
+Have a look at the [Faikin](https://github.com/revk/ESP32-Faikin) project for an alternative for thse units.
```

Comment:
While we're here... The above part "(AC only)" possibly isn't correct, as many of those units can heat and cool too... Maybe "Other Daikin units" or something could make sense

Reply:
But they can do A2A heating and cooling...